onnx/reference/ops/aionnxml/_op_list.py

```diff
@@ -5,9 +5,8 @@
 # probabilites not consumed by any operator.
```

Comment:
Although not relevant to this PR, this misspell is good to be fixed.

Reply:
Good to see you @jcwchen :) Done. Thanks!